.github/workflows/release.yaml

```diff
@@ -66,6 +66,11 @@ jobs:
           severity: 'CRITICAL,HIGH'
           format: 'sarif'
           output: 'trivy-results-${{ matrix.platforms }}.sarif'
+        env:
+          # Use AWS' ECR mirror for the trivy-db image, as GitHub's Container
+          # Registry is returning a TOOMANYREQUESTS error.
+          # Ref: https://github.com/aquasecurity/trivy-action/issues/389
+          TRIVY_DB_REPOSITORY: 'public.ecr.aws/aquasecurity/trivy-db:2'
```

Comment:
Is there any restriction on `public.ecr.aws/aquasecurity/trivy-db`? @dims @chaochn47

Reply:
I thought it was okay, as there are [many instances](https://github.com/search?q=repo%3Akubernetes%2Ftest-infra%20public.ecr.aws&type=code) of images from the `public.ecr.aws` registry in the k/test-infra repo, including our website listing, as it uses [an AWS ECR mirror](https://github.com/kubernetes/test-infra/blob/master/config/jobs/etcd/etcd-website-presubmits.yaml#L16) of the NodeJS image.

Either way, it'd be great to have the blessing to use it :v:.